github/compliance/report.py

```diff
@@ -393,8 +385,17 @@ def _template_vars(
     elif issue_type == _compliance_label_malware:
         template_variables |= _malware_template_vars(result_group)
 
-    elif issue_type == _compliance_label_cfg_policy_violation:
-        template_variables |= _cfg_policy_violation_template_vars(result_group)
+    elif issue_type in (
+        _compliance_label_credentials_outdated,
+        _compliance_label_no_responsible,
+        _compliance_label_no_rule,
+        _compliance_label_no_status,
+        _compliance_label_undefined_policy,
+    ):
+        template_variables |= _cfg_policy_violation_template_vars(
```

Comment:
if this group of issue-types shares commit template-vars, define a dedicated function for those, rather than passing-ing issue-type. or just add the one additional template-var at the beginning. do we actually need this as template-var? why not simply add to the type-specific template?